docs/contributing/running-and-writing-acceptance-tests.md

```diff
@@ -502,11 +502,34 @@ resource "aws_example_thing" "test" {
 }
 ```
 
-#### Service Availability PreCheck
+#### PreChecks
```

Comment:
The table of contents needs to be updated. 👍

Reply:
Updated!